makefiles/kconfig.mk

```diff
@@ -72,13 +75,16 @@ SHOULD_RUN_KCONFIG ?= $(or $(wildcard $(APPDIR)/*.config), \
                            $(if $(CLEAN),,$(wildcard $(KCONFIG_OUT_CONFIG))), \
                            $(filter menuconfig, $(MAKECMDGOALS)))
 
+# export variable to make it visible in other Makefiles
+export SHOULD_RUN_KCONFIG
+
 ifneq (,$(SHOULD_RUN_KCONFIG))
 
 # Flag to enable the --sync-dir feature of Kconfiglib
 KCONFIG_SYNC_DEPS ?=
```

Comment:
This is no longer used, you can remove it.

Reply:
amended the change